### PR TITLE
Performance testing

### PR DESCRIPTION
**Pull request by 6l20**
## PR description

- Automate performance testing
- Discard private-tx testing

## Documentation

- [ ] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [ ] I thought about the changelog and included a [changelog update if required](https://github.com/consensys/orchestrate/blob/master/CHANGELOG.md).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -300,6 +300,25 @@ jobs:
             KAFKA_CONSUMER_GROUP_NAME: "stress"
           command: ./build/bin/test stress
 
+  run-perf:
```
Comment (ggarri) [inline .circleci/config.yml:303]: avoid no standardized acronyms so it is easy to read

File: Makefile [hunk 2]
```diff
@@ -114,7 +114,7 @@ lint-tools: ## Install linting tools
 tools: lint-tools ## Install test tools
 	go install github.com/golang/mock/mockgen@v1.4.3
 	go install github.com/swaggo/swag/cmd/swag@v1.7.8
-	go install github.com/tsenart/vegeta
+	go get -u github.com/tsenart/vegeta
```
Comment (ggarri) [inline Makefile:117]: `go get` install dependencies locally on the project. In this case we want vegeta to be installed globally

Comment (6l20) [reply Makefile:117]: go install fails with
`no required module provides package github.com/tsenart/vegeta`
And go get adds dependencies indeed.

Comment (ggarri) [reply Makefile]: Then use precompile ones https://github.com/tsenart/vegeta/releases

**Pull request merged**